nbdime/nbmergeapp.py

```diff
@@ -48,12 +48,12 @@ def main_merge(args):
     else:
         nbdime.log.debug("Merge completed successfully with no unresolvable conflicts.")
 
-    if args.decisions:
+    if decisions:
```

Comment:
No, args.decisions is deliberate, it shouldn't print decisions without --decisions on the commandline.